llvm/include/llvm/CAS/CASOutputBackend.h

```diff
@@ -11,8 +11,11 @@
 
 #include <llvm/Support/Error.h>
 #include <llvm/Support/OutputBackend.h>
+#include <memory>
```

Comment:
Is this for `std::shared_ptr<>` below? I suggest just pushing this separately (no need for a review).